admin/src/components/Title.js

```diff
@@ -74,14 +114,22 @@ function TitleTableRow({ data, onShow, onEdit, onDelete }) {
             <ShowPresetButton onClick={onShow} active={data.shown}/>
         </TableCell>
         <PresetsTableCell value={data.settings.preset} editValue={editData?.settings?.preset} isActive={data.shown}
-            onChangeValue={onChangeFieldHandler(setEditData, "preset")} onSubmitAction={onSubmitEdit}/>
+            onChangeValue={onChangeFieldHandler(setEditData, "preset")} onSubmitAction={onSubmitEdit}
+            ValueEditor={TemplateEditor}
```

Comment:
codestyle: props should start with a lowercase letter

Reply:
If you want to then use it as a JSX tag use named object dereferencing
Like this:
```javascript
const PresetTableCell = ({valueEditor: ValueEditor, ...props}) => {
// ...
}
```